### PR TITLE
ECL-194: Send registration submitted emails to primary and secondary contact

### PR DESCRIPTION
**Pull request by peter-hazell**
- This should not be merged until https://jira.tools.tax.service.gov.uk/browse/DC-4733 has been actioned by the DC team and https://github.com/hmrc/hmrc-email-renderer/pull/1001 has been merged and deployed to QA.

### CODE REVIEW

File: test/uk/gov/hmrc/economiccrimelevyregistration/connectors/EmailConnectorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        val response = HttpResponse(ACCEPTED, "")
+
+        when(
+          mockHttpClient
+            .POST[RegistrationSubmittedEmailRequest, Either[UpstreamErrorResponse, HttpResponse]](
+              ArgumentMatchers.eq(expectedUrl),
+              ArgumentMatchers.eq(
+                RegistrationSubmittedEmailRequest(
+                  Seq(to),
+                  templateId = "ecl_registration_submitted",
```
Comment (TonyCOgilvie) [inline test/uk/gov/hmrc/economiccrimelevyregistration/connectors/EmailConnectorSpec.scala:49]: Suggest move the email templateId into an object with a const val.

File: test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala [hunk 2]
```diff
@@ -56,4 +59,34 @@ class ViewUtilsSpec extends SpecBase {
...
+
+    "correctly format a non-translated local date" in {
+      val localDate = LocalDate.parse("2007-12-03")
+
+      ViewUtils.formatLocalDate(localDate, translate = false)(messages) shouldBe "3 December 2007"
+    }
+  }
+
+  "formatMoney" should {
+    "format a moneyary amount with commas" in {
```
Comment (TonyCOgilvie) [inline test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala:85]: typo in description

File: test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala [hunk 3]
```diff
@@ -56,4 +59,34 @@ class ViewUtilsSpec extends SpecBase {
     }
   }
 
+  "formatDate" should {
```
Comment (TonyCOgilvie) [inline test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala:62]: Is there any benefit in specifying a date that is unambiguous for these tests, e.g. 2007-12-31?

**Review by TonyCOgilvie — COMMENTED** [review]
See individual comments

File: test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala (hunk 3 above)
Comment (peter-hazell) [reply test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala:62]: I deliberately chose the date 3 December 2007 as it proves that leading zeros aren't displayed in the output. I think the ambiguity in the parsing of the string into a LocalDate isn't an issue as that's not really what we're testing, the expected output makes it clear enough I think.

**Review by TonyCOgilvie — APPROVED** [review]

**Pull request merged**